gui/wxpython/datacatalog/tree.py

```diff
@@ -205,17 +205,19 @@ def __init__(self, data=None):
 
     @property
     def label(self):
-        if self.data["type"] == "mapset":
-            if self.data["lock"] and self.data["user"]:
-                return self.data["name"] + " <user: {}>".format(self.data["user"]) + " (locked)"
-            elif self.data["lock"]:
-                return self.data["name"] + " (locked)"
-            elif self.data["user"]:
-                return self.data["name"] + " <user: {}>".format(self.data["user"])
+        if self.data['type'] == 'mapset':
+            if self.data['current']:
+                return self.data['name'] + _(" (current)")
```

Comment:
More general way to do this is:


```
_("{name} (current)").format(name=data["name"])
```

or making use of the fact data is a dictionary:

```
_("{data[name]} (current)").format(data=data)
```

or making use of the fact that the only values we are using are from the dictionary:


```
_("{name} (current)").format(**data)
```

It is more flexible and possibly also more readable as you see the basic structure of the string being constructed, but in the other cases it is also likely faster since format is already used there anyway.

Reply:
Rewritten to _("{name} (current)").format(**data).